engine/table/build.gradle

```diff
@@ -42,7 +42,7 @@ dependencies {
 
     Classpaths.inheritGroovy(project, 'groovy', 'implementation')
     Classpaths.inheritGroovy(project, 'groovy-json', 'implementation')
-    Classpaths.inheritJavaParser(project, 'compile')
+    Classpaths.inheritJavaParser(project, 'api')
```

Comment:
Why not `implementation`?

Reply:
```
io/deephaven/server/table/validation/ColumnExpressionValidator.java:6: error: package com.github.javap
arser does not exist                                                                                                                                                
import com.github.javaparser.JavaParser;                        
```

Reply:
Even though server uses it as part of "public" api, we really should eventually refactor that - it's really an implementation detail, so I have it explicitly as implementation now for server.